advanced/editor-command-identifiers.md

```diff
@@ -7,93 +7,397 @@ description: The complete list of exposed editor commands.
+| mceApplyTextcolor  | Applies text color or background color to the current selection. Requires an argument of either `'hilitecolor'` or `'forecolor'`, and the value of the color. |
+| mceRemoveTextcolor | Removes the text color or background color from the current selection. Requires an argument of either `'hilitecolor'` or `'forecolor'`.                       |
+| RemoveFormat       | Removes any formats from the current selection.                                                                                                               |
+| mceBlockQuote      | Wraps the selected text blocks into a block quote.                                                                                                            |
+| FormatBlock        | Changes the format of the current block. The value passed in should be the block name. For example: `H1`, `bold`, `code`, or `P`.                             |
+| mceInsertContent   | Inserts contents at the current selection. The value passed in should be the contents to be inserted.                                                         |
+| mceReplaceContent  | Replaces the current selection. The value passed in should be the new content.                                                                                |
+| mceSetContent      | Sets the contents of the editor. The value is the contents to set as the editor contents.                                                                     |
+| mceToggleFormat    | Toggles a specified format by name. The value is the name of the format to toggle.                                                                            |
+| ToggleSidebar      | Closes the currrent sidebar, or toogles the sidebar if the sidebar name is provided as a value (_`<sidebar-name>`_).                                          |
+| Indent             | Indents the current selection.                                                                                                                                |
+| Outdent            | Outdents the current selection.                                                                                                                               |
+| InsertLineBreak    | Adds a line break `<br/>` at the current cursor or selection.                                                                                                 |
+| mceInsertNewLine   | Adds a new line at the current cursor or selection, such as splitting the current paragraph element.                                                          |
+| mceInsertRawHTML   | Inserts the RAW HTML passed as a value, overwriting the current selection or at the cursor position.                                                          |
+| mceToggleVisualAid | Toggles the visual aids for tables without borders etc.                                                                                                       |
```

Comment:
This will also toggle the anchor/bookmark icon fwiw.

Reply:
Fixed